docs/Makefile

```diff
@@ -44,7 +44,7 @@ help:
 clean:
 	-rm -rf $(BUILDDIR)
 
-html:
+html: clean
```

Comment:
When testing the results with `make html`, some cache it was giving me some trouble. And because of that the toctree was not being generated correctly, even after forcing the refresh and cleaning the cache on the browser.
`make clean` fixed the problem.

But yes, I agree. I will split this.